app/scripts/app.js

```diff
@@ -61,5 +75,46 @@ angular
       replace: true,
       templateUrl: 'views/alert.html'
     };
-  }]);
-    
\ No newline at end of file
+  }])
```

Comment:
@rassiel @kpunjani 
The following two directives allow for validating a form field for date and time. Well, I haven't tested the time one yet. Date works though. 

I have not looked into it yet - but is it possible to put these directives into separate files, for better management and maintenance?